ibm/service/power/data_source_ibm_pi_instances.go

```diff
@@ -273,7 +273,7 @@ func flattenPvmInstanceNetworks(list []*models.PVMInstanceNetwork) (networks []m
 			p := make(map[string]interface{})
 			p[Attr_ExternalIP] = pvmip.ExternalIP
 			p[Attr_IP] = pvmip.IPAddress
-			p[Attr_MacAddress] = pvmip.MacAddress
+			p[Attr_Macaddress] = pvmip.MacAddress
```

Comment:
isn't Attr_Macaddress deprecated? should it be Attr_MacAddress?

Reply:
Good catch! Both need to set here, will be making another PR to remove them later.